gatsby/content/blog/2023/01/2023-01-31-synapse-1.76.mdx

```diff
@@ -0,0 +1,85 @@
+---
+date: '2023-01-31'
+title: Synapse 1.76 released
+categories:
+- Releases
+author: H. Shay
+---
+
+Greetings matrix fans! We've published Synapse [version 1.76](https://github.com/matrix-org/synapse/releases/tag/v1.76.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience. Please take a look at the [upgrade notes](https://matrix-org.github.io/synapse/v1.76/upgrade.html#upgrading-to-v1760)
+for any important information about upgrading.
+
+## Announcements
+
+The big news is that faster joins for all has arrived! When joining a room for the first time,
+Synapse 1.76.0 will request a partial join from the other server by default. Previously, server admins
+had to opt-in to this using an experimental config flag.
+
+Server admins can opt out of this feature for the time being by setting
+
+```
+experimental:
+    faster_joins: false
+```
+
+in their server config.
+
+For everyone else, after a faster join Synapse considers that room "partially joined". In this state, you should be able to
+
+- read incoming messages;
+- see incoming state changes, e.g. room topic changes; and
+- send messages, if the room is unencrypted.
+
+Synapse has to spend more effort to complete the join in the background. Once this finishes, you will be able to
+
+- send messages, if the room is encrypted;
+- retrieve room history from before your join, if permitted by the room settings; and
+- access the full list of room members.
+
+Practically, this means that the experience of joining a large room over federation should be greatly improved. In past
+versions of Synapse, joining a large room such as Matrix HQ over federation could take up to 30 mins. This time has been
+cut to ~20-30 seconds in the new release. Those of you who have been watching this know this has been a huge undertaking,
```

Comment:
This kind of sets the wrong expectation a bit, I'd try to be more careful so that people know that the room is only *partially* joined after 20-30 seconds, but it's not quite the real deal.
(I've seen people in community rooms confused about this — 'faster joins are a bit slow' etc etc)

Reply:
I've changed it to reflect partially joining.